### DISCUSSION TITLE
I can visit github.com but I can't singin github account as active copilot(i had a copilot subscription)

### DISCUSSION THREAD

**Discussion by hickiy** [community/community#64494] [Copilot Conversations]
### Select Topic Area

Bug

### Body

[ERROR] [default] [2023-08-21T12:24:07.611Z] GitHub Copilot could not connect to server. Extension activation failed: "network error"

vscode and  plateform infomation 
Version: 1.81.1 (Universal)
Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794
Date: 2023-08-09T22:20:33.924Z
Electron: 22.3.18
ElectronBuildId: 22689846
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.6.0

copilot version v1.105.350

copilot chat version v0.6.0

**Comment by arafajs** [comment]
- Make sure that you are connected to the internet.
- Try restarting your computer and VS Code.
- Reinstall the GitHub Copilot extension.
- Check your firewall settings to make sure that GitHub Copilot is allowed to access the internet.

Here are some additional things to consider when troubleshooting this issue:

- The GitHub Copilot API is currently in beta, so there may be some temporary outages or errors.
- If you are using a proxy server, you may need to configure your proxy settings in VS Code.
- If you are using a VPN, you may need to disable your VPN to connect to the GitHub API.
- If you are still having trouble signing in to GitHub Copilot, you can contact GitHub Copilot support for help.
